src/firestore/index.ts

```diff
@@ -96,13 +110,29 @@ export {
  *   service if no app is provided or the `Firestore` service associated with the
  *   provided app.
  */
-export function getFirestore(app?: App): Firestore {
-  if (typeof app === 'undefined') {
-    app = getApp();
-  }
-
+export function getFirestore(app: App): Firestore;
+/**
+ * @param databaseId
+ * @internal
+ */
+export function getFirestore(databaseId: string): Firestore;
+/**
+ * @param app
+ * @param databaseId
+ * @internal
```

Comment:
`@internal` will hide this from both reference docs and autocomplete (it will remove the typescript annotation). Is that what we expect here? If we only want to hide from reference docs and would like to keep autocomplete we can use `@alpha` instead.